src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2260,7 +2268,7 @@ namespace winrt::TerminalApp::implementation
 
     bool TerminalPage::FocusMode() const
     {
-        return _isInFocusMode;
+        return _isInFocusMode || IsQuakeWindow();
     }
```

Comment:
This seems a bit weird to me. If QM is always in FM, why do we need to make this change at all? Shouldn't `_isInFocusMode` be updated to be `true` when in QM, and `false` when out of it?

Reply:
yea that's weird